server/components/email/index.js

```diff
@@ -14,8 +15,27 @@ module.exports.signup = function(message, callback){
   email.addSubstitution("-name-", message.name); //sub. variables
   email.addSubstitution("-verifyURL-", message.verifyURL); //sub. variables
   sendEmail(email, callback)
+  if(process.env.NODE_ENV != "production") {
```

Comment:
we need a more permanent solution in place eventually, but this works well for now IMO